src/Security/Auth0EntryPoint.php

```diff
@@ -45,15 +43,15 @@ public function start(Request $request, AuthenticationException $authException =
         $csrfToken = $this->csrfTokenManager->getToken('auth0-sso');
 
         $query = [
-            'response_type' => 'code',
-            'client_id' => $this->auth0ClientId,
-            'redirect_uri' => $this->httpUtils->generateUri($request, $this->callbackRoute),
+            'response_type' => $this->configuration->getResponseType(),
+            'client_id' => $this->configuration->getClientId(),
+            'redirect_uri' => $this->httpUtils->generateUri($request, $this->loginCheckRoute),
```

Comment:
@Nyholm I think we must use the redirectUri issued from the `SdkConfiguration`, otherwise the parameter will be never, this could be crazy to users to understand why :exploding_head:

Reply:
We can't use `redirectUri`. If we did, the user has to hard code the URL instead of using symfony's routing system. 
It would be a lot more hassle with dev/qa/prod environment etc.

Reply:
Maybe we can take the route in this configuration, and call the url generator from the route ?

Reply:
But that would not be less confusing =)

I suggest keeping this as it is atm. We can improve that in the next PR.

Reply:
But that would not be less confusing =)

Reply:
OK for the moment, will think about it later !